lib/rules/no-dupe-keys.js

```diff
@@ -6,6 +6,79 @@
 "use strict";
 
 //------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const astUtils = require("../ast-utils");
+
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+const GET_KIND = /^(?:init|get)$/;
```

Comment:
Normal properties' `kind` is `"init"`. This regexp is to address normal properties as getter/setter pair. As a result, this rule will address `{a: 1, get a() {}}` as duplicate keys. This is the purpose of this PR :)